src/clarity-angular/datagrid/built-in/filters/datagrid-string-filter.ts

```diff
@@ -77,8 +77,12 @@ export class DatagridStringFilter extends DatagridFilterRegistrar<DatagridString
     /**
      * Common setter for the input value
      */
+    public get value() {
+        return this.filter.value;
+    }
     @Input("clrFilterValue")
     public set value(value: string) {
```

Comment:
You HTML `<input>` still has `[(ngModel)]="filter.value"`. So if I type something in it, it will update `this.filter.value`, not `this.value`.  Which means you "bypass" this setter, so the `filterValueChange` output is never emitted.
All you need to do to fix that is switch to `[(ngModel)]="value"`.

I'm still figuring out what the best pattern for our unit tests is, because this could have slipped through while the tests do pass. And manually grabbing a native input element and typing in it in Jasmine is really something I would not want to do in every test we have. Until we figure out a solution, the change I mentioned in this comment is good enough for me.